ui/desktop/electron-app/src/index.js

```diff
@@ -113,11 +113,18 @@ app.on('ready', async () => {
   }
   Menu.setApplicationMenu(menuTemplate);
 
+  /**
+   * Enable electron OS window frame/chrome for MacOS only.
+   * Disable frame/chrome regardless of OS when `DISABLE_WINDOW_CHROME` is set.
+   */
+  let showWindowChrome = isMac();
+  if (process.env.DISABLE_WINDOW_CHROME) showWindowChrome = false;
```

Comment:
Curiosity question, when we use `DISABLE_WINDOW_CHROME` should we just define it? or also give it a value? I mean, on the terminal, when executing desktop app, should we do `$ DISABLE_WINDOW_CHROME=true yarn start:desktop` or `$ DISABLE_WINDOW_CHROME yarn start:desktop` ?

Reply:
Good point. Checking for `DISABLE_WINDOW_CHROME` truthy value is clearer. Will update.

Reply:
Made minor modification to comment text to specify that `DISABLE_WINDOW_CHROME=true` is used.